### ISSUE TITLE
Is it possible to use this repo to fine tune on other instance segmentation tasks? 

### ISSUE THREAD

**Issue by stephenllh** [microsoft/SoftTeacher#82]

**Comment by MendelXu** [comment]
It is determined by what the network it is, for the two-stage detectors, I think it is not hard to adopt such change. For single-stage one, I think it is difficult to do that in a short time.

**Comment by stephenllh** [comment]
How about MaskRCNN?

**Comment by MendelXu** [comment]
In the paper, only the HTC detector has a mask branch.  
In the case of MaskRCNN, you can load a trained model and finetune it with our approach, and with shorter iterations.

**Comment by stephenllh** [comment]
I see. Very much thanks!
